<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Create_Asset_Db_Dialog</class>
 <widget class="QDialog" name="Create_Asset_Db_Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>199</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>740</width>
    <height>199</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>740</width>
    <height>199</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create_Asset_Db_Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>160</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>701</width>
     <height>122</height>
    </rect>
   </property>
   <property name="title">
    <string>Asset Database Parameters</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_SelectPathToSrcAssets">
      <property name="text">
       <string>Select Path to Source Assets</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_pathToSrcAssets"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_SelectPathToIntAssets">
      <property name="text">
       <string>Select Path to Intermediate Assets</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_pathToIntAssets"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_SelectPathToBinAssets">
      <property name="text">
       <string>Select Path to Compiled Assets</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_pathToBinAssets"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
